docs/reference-sequence-import.md

```diff
@@ -0,0 +1,124 @@
+# Importing Reference Sequence
+
+ This document shows the flow of how different types of ReferenceSequences are extracted and imported.
```

Comment:
```suggestion
This document shows the flow of how different types of ReferenceSequences are extracted and imported.
```

I don't think we use indentation inside paragraphs, could you please fix this throughout the document?

Reply:
fixed in f3f80b9